journal/modernized/01-journal.adoc

```diff
@@ -383,11 +383,12 @@ that I read of in the scripture; and the great Creator said,
 "`Every thing that he had made was very good,`" Gen. 1:31.,
 and his language to man was very good and pure.
 Then again, when Adam transgressed the law and commandment of God,
-"`the Lord God called unto Adam, and said unto him.
-Where art thou?
+"`the Lord God called unto Adam, and said unto him,
+"`Where art thou?
 And he said, I heard thy voice in the garden, and I was afraid.
+because I was naked,`" Gen. 3:10-9. 
```

Comment:
I think the quotation mark you added here, should either a) be left out, or b) be switched to a single-quote, and wrapped around the rest of the _inner quote_, since this is a quote within a quote (starting on line 386). So, what I would propose is probably b), which would be:

```ASCIIDOC
"`the Lord God called unto Adam, and said unto him,
'`Where art thou?`'
And he said, '`I heard thy voice in the garden, and I was afraid
because I was naked,`'`" Gen. 3:10-9. 
```

Note the single quotes wrapping _Where art thou?_ and _I heard thy voice in the garden, and I was afraid because I was naked._ 

Also note, I think we should remove the ending period on line 388, that looks like a mistake to me.  Maybe a comma, but definitely not a period, and dropping it seems fine to me.